### ISSUE TITLE
sequential iteration runner

### ISSUE THREAD

**Issue by RustedAperture** [usebruno/bruno#3261]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

I would like to run iterations sequentially vs in parallel to prevent 429 errors. I do not have this issue in postman when running using a large csv since their delay and runner appear to run sequentially. Each iteration is run one after the other rather than all at the same time.

### Mockups or Images of the feature

perhaps just a checkbox on both folder runner (right click run) and the collection runner (top right) to enable sequential running.

**Comment by helloanoop** [comment]
Closing this as this is duplicate of #3296
